lib/rules/comment-whitespace-inside/index.js

```diff
@@ -59,7 +59,7 @@ const rule = (primary, _secondaryOptions, context) => {
 			}
 
 			const rawComment = comment.toString();
-			const firstFourChars = rawComment.substr(0, 4);
+			const firstFourChars = rawComment.slice(0, 4);
```

Comment:
[note] The second argument of `substr()` means a *length*, while the one of `slice()` means an *endIndex*. If the first argument is `0`, they can be easily replaced.